docs/src/auth.md

```diff
@@ -47,7 +47,7 @@ Create `tests/auth.setup.ts` that will prepare authenticated browser state for a
 
 ```js title="tests/auth.setup.ts"
 import { test as setup, expect } from '@playwright/test';
-import path from 'path';
```

Comment:
That seems backwards to me

Reply:
import * as path from 'path' is the right way!